src/current/v24.3/ldap.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Configure LDAP Authentication and Authorization
+summary: A guide to the configuration of authentication and authorization using LDAP or Active Directory.
+toc: true
+---
+
+{% include enterprise-feature.md %}
+
+CockroachDB supports authentication and authorization using LDAP (Lightweight Directory Access Protocol) or Active Directory servers. This allows you to integrate your existing directory service with CockroachDB for centralized user management and access control.
+
+Q: Should we be flipping these and putting authorization before authentication?  
+Q: Can customers just use one, if they want, of LDAP authn and authz? What do we expect will be typical or recommended?  
+Q: Decide if Authz and Authn should be 2 separate pages.  
```

Comment:
I will leave it you to decide, if it needs to be two different pages. But, in long term, I feel it might be a good idea to have a separate section for authorization with different pages for each mechanism exactly like authentication. Today, authorization is only supported by LDAP/AD.